app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -385,15 +417,6 @@ fun MapScreen(
       locationPermissionState = locationPermissionState,
       context = context)
 
-  DisposableEffect(Unit) {
-    onDispose {
-      LocationUtils.stopUserLocationUpdates(context, locationUpdatedCallback)
-      mapView.overlays.clear()
-      mapView.onPause()
-      mapView.onDetach()
-    }
-  }
-
```

Comment:
Question: Why have you removed this part ?

Reply:
It was a duplicate! The same effect already existed above!

Reply:
Good catch!